No LICENSE and README.md into the top level site-packages.

diff -Nrup a/pyproject.toml b/pyproject.toml
--- a/pyproject.toml	2026-03-18 20:44:58.000000000 +0100
+++ b/pyproject.toml	2026-10-22 17:22:15.719778374 +0200
@@ -59,5 +59,3 @@ include = [
 "mtk.py" = "mtkclient/mtk.py"
 "mtk_gui.py" = "mtkclient/mtk_gui.py"
 "stage2.py" = "mtkclient/stage2.py"
-"LICENSE" = "LICENSE"
-"README.md" = "README.md"
